service/Microsoft.Azure.Devices/ServiceClient.cs

```diff
@@ -88,8 +88,9 @@ public static ServiceClient CreateFromConnectionString(string connectionString,
         /// </summary>
         /// <param name="deviceId">The device identifier for the target device</param>
         /// <param name="message">The message containing the notification</param>
+        /// <param name="timeout">The operation timeout</param>
```

Comment:
For AMQP this is actually the operation timeout override. If not used, the `OperationTimeout` is used instead. Docs should point that out.